cromshell

```diff
@@ -88,12 +88,14 @@ function turtle()
 
 function turtleDead()
 {
-  error "    ,,    ,,     ,,       "
-  error "    \\‾\\,-/‾/====/‾/,     "
-  error "     \\/=<‾><‾><‾><‾>-,   "
-  error "     / (\\‾\\‾|‾/‾/‾/‾     "
-  error "    \\‾x/ ˙'-;-;-'˙       "
-  error "     ‾‾                  "
+  export RED='\033[0;31m'
```

Comment:
Can you remove the `RED` and `NC` variables and replace them with the global colors above?

Specifically, `RED` should be `COLOR_FAILED` and `NC` should be `COLOR_NORM`.

Reply:
Just updated as you suggested, and if this is what you meant (background color red)

<img width="655" alt="screen shot 2019-02-25 at 12 23 48 pm" src="https://user-images.githubusercontent.com/16310888/53355964-44461800-38f8-11e9-8298-d37d424c3a6b.png">

<img width="656" alt="screen shot 2019-02-25 at 12 26 03 pm" src="https://user-images.githubusercontent.com/16310888/53356052-8a9b7700-38f8-11e9-84b6-8fad3d8349c5.png">

Reply:
Yup - I figured it would be fine to reuse the color that was already there.